### DISCUSSION TITLE
Disabling auto onboarding for our Org

### DISCUSSION THREAD

**Discussion by hskrtich** [renovatebot/renovate#17134] [Request Help]
Hi, I read here https://github.com/renovatebot/config-help/issues/280 that auto onboarding can be disabled for the github app by support. Can someone help me do that for our org?

https://github.com/ThirtyMadison

Thank you!

**Comment by rarkins** [comment]
Hi, to make sure there's no misunderstanding, can you clarity what behavior you'd want? e.g. either:

- Just run on every repo with default settings, or
- Don't run on any repo unless you push a config file manually?

**Comment by hskrtich** [comment]
I am looking for "Don't run on any repo unless you push a config file manually"

**Reply by rarkins** [reply]
I've now set `"onboarding":false` for your org. Won't affect any existing repos which have onboarding PRs, but if you add a new repo (I suggest using a test one) then you should see that it doesn't onboard.

**Comment by hskrtich** [comment]
Hi, for some reason we are getting the onboarding PR's again. Can someone check on this?

**Reply by rarkins** [reply]
Sorry, I will look into it

**Reply by hskrtich** [reply]
Thank you!

**Reply by hskrtich** [reply]
Any updates on this?

**Reply by rarkins** [reply]
Sorry for not updating, it was fixed that day and shouldn't be happening anymore.
